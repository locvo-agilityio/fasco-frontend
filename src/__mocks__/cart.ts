import { ICart } from '@/types';

export const MOCK_CARTS: ICart[] = [
  {
    id: '1',
    name: 'Sofa',
    quantity: 1,
    price: 9550,
    image:
      'https://down-vn.img.susercontent.com/file/vn-11134207-7ras8-m3sbmx26g0cc16.webp',
    color: 'White',
    totalQuantity: 5,
    size: 'M',
    total: 9550,
    productId: '',
  },
  {
    id: '2',
    name: 'Sofa',
    quantity: 1,
    price: 9550,
    image:
      'https://down-vn.img.susercontent.com/file/vn-11134207-7ras8-m3sbmx26g0cc16.webp',
    color: 'Red',
    totalQuantity: 5,
    size: 'L',
    total: 9550,
    productId: '',
  },
];
