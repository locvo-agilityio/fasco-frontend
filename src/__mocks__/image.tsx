export const MOCK_IMAGE_GALLERY = [
  {
    id: 1,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134201-7rdxx-lz73hliz42dj59.webp',
  },
  {
    id: 2,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rdxl-lydeljvb148a71.webp',
  },
  {
    id: 3,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rdwt-lydeljvb144o82.webp',
  },
  {
    id: 4,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rd70-lu7m75qrwqihda.webp',
  },
  {
    id: 5,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rd6a-lu7m75szs3i1cc.webp',
  },
  {
    id: 6,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rd4m-lu7m75sfswqn05.webp',
  },
  {
    id: 7,
    image:
      'https://down-vn.img.susercontent.com/file/sg-11134301-7rd73-lu7m75tjr9rted.webp',
  },
];
