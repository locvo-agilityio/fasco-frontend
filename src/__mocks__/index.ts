export * from './next-auth';
export * from './next-auth-providers-credentials';
export * from './image';
export * from './cart';
