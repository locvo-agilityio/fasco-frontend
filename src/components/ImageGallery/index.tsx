'use client';

import { useState } from 'react';
import Image from 'next/image';
import { cn } from '@/utils';

interface ImageGalleryProps {
  images: { id: number; image: string }[];
}

const ImageGallery = ({ images }: ImageGalleryProps) => {
  const [selectedImage, setSelectedImage] = useState(images[0].image);

  const handleImageClick = (img: string) => {
    setSelectedImage(img);
  };

  return (
    <div className="flex flex-col md:flex-row items-center md:items-start gap-5">
      <div className="flex md:flex-col gap-6">
        {images.map(({ id, image }) => {
          const isSelectImage =
            selectedImage === image ? 'border-2 border-blue-500' : '';

          return (
            <div
              key={id}
              className={cn(
                'w-14 h-[77px] border overflow-hidden cursor-pointer relative',
                isSelectImage,
              )}
              onClick={() => handleImageClick(image)}
            >
              <Image
                src={image}
                alt={`Thumbnail ${id}`}
                fill
                style={{ objectFit: 'cover' }}
                sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
                priority
              />
            </div>
          );
        })}
      </div>

      <div className="w-[491px] h-[654px] border overflow-hidden relative">
        <Image
          src={selectedImage}
          alt="Selected"
          fill
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          priority
          style={{ objectFit: 'cover' }}
        />
      </div>
    </div>
  );
};

export default ImageGallery;
