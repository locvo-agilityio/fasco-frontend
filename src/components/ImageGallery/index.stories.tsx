import type { Meta, StoryObj } from '@storybook/react';

// Components
import ImageGallery from '.';

// Mocks
import { MOCK_IMAGE_GALLERY } from '@/__mocks__';

const meta = {
  title: 'Components/ImageGallery',
  component: ImageGallery,
  parameters: {
    layout: 'centered',
  },
  tags: ['autodocs'],

  render: (args) => (
    <div className="w-[1280px]">
      <ImageGallery {...args} />
    </div>
  ),
} satisfies Meta<typeof ImageGallery>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    images: MOCK_IMAGE_GALLERY,
  },
};
