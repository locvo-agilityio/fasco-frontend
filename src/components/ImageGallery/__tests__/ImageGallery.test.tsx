import { fireEvent, render, RenderResult } from '@testing-library/react';

// Components
import ImageGallery from '..';

// Mocks
import { MOCK_IMAGE_GALLERY } from '@/__mocks__';

let renderResult: RenderResult;

describe('ImageGallery', () => {
  beforeEach(() => {
    renderResult = render(<ImageGallery images={MOCK_IMAGE_GALLERY} />);
  });

  it('should render correctly', () => {
    expect(renderResult.container).toMatchSnapshot();
  });

  it('handle click on image', () => {
    const { getByAltText } = renderResult;

    const image = getByAltText('Thumbnail 1');

    fireEvent.click(image);

    expect(image).toBeInTheDocument();
  });
});
