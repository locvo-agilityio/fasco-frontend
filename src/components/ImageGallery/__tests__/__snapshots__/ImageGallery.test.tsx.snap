// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageGallery should render correctly 1`] = `
<div>
  <div
    class="flex flex-col md:flex-row items-center md:items-start gap-5"
  >
    <div
      class="flex md:flex-col gap-6"
    >
      <div
        class="w-14 h-[77px] overflow-hidden cursor-pointer relative border-2 border-blue-500"
      >
        <img
          alt="Thumbnail 1"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 2"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdxl-lydeljvb148a71.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 3"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rdwt-lydeljvb144o82.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 4"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd70-lu7m75qrwqihda.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 5"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd6a-lu7m75szs3i1cc.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 6"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd4m-lu7m75sfswqn05.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
      <div
        class="w-14 h-[77px] border overflow-hidden cursor-pointer relative"
      >
        <img
          alt="Thumbnail 7"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
          src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=3840&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134301-7rd73-lu7m75tjr9rted.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
        />
      </div>
    </div>
    <div
      class="w-[491px] h-[654px] border overflow-hidden relative"
    >
      <img
        alt="Selected"
        data-nimg="fill"
        decoding="async"
        sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
        src="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=3840&q=75"
        srcset="/_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fdown-vn.img.susercontent.com%2Ffile%2Fsg-11134201-7rdxx-lz73hliz42dj59.webp&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
      />
    </div>
  </div>
</div>
`;
